### PR TITLE
Update MMS NoW ETLs

### PR DESCRIPTION
**Pull request by nayr974**

### CODE REVIEW

File: tasks/now-etls/mms_now_import.py
Comment (exzizt) [inline tasks/now-etls/mms_now_import.py:47]: - 'ACCEPTED' THEN 'Pending Verification'?

- Client delayed --> Client Delayed

Comment (mmumby) [reply tasks/now-etls/mms_now_import.py:47]: We decided yesterday that 'PERMIT CLOSED'  -> Approved (It was once approved, and then they mixed in the permit logic and set to permit closed when the bond is returned)

Comment (mmumby) [reply tasks/now-etls/mms_now_import.py:47]: We also don't have  'Pending Approval' as an option any longer,
 'PENDING APPROVAL' THEN 'Pending Verification' ? 
 Users will have to import and start the application from scratch within CORE

Comment (nayr974) [reply tasks/now-etls/mms_now_import.py:47]: "Client delayed" matches the value in our code table. Accepted -> pending verification per Ashley's direction. Updated permit closed to approved.

Comment (exzizt) [reply tasks/now-etls/mms_now_import.py]: It was changed to have consistent casing with my PR yesterday.

Comment (nayr974) [reply tasks/now-etls/mms_now_import.py]: Ugh, FINE Luke

Comment (exzizt) [reply tasks/now-etls/mms_now_import.py]: ;)

**Review by exzizt — APPROVED** [review]
💥

**Pull request closed without merging**
